spec/isolation/components/mailer_configuration/without_hanami_mailer_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe "Components: mailer.configuration", type: :cli do
+  context "without hanami-mailer" do
+    it "resolves mailer configuration" do
+      with_project do
+        environment_file = Pathname.new(Dir.pwd).join("config", "environment")
+        replace "#{environment_file}.rb", "delivery :test", ""
```

Comment:
Same of https://github.com/hanami/hanami/pull/742/commits/5523b86ca13a8f3145f3a41257b1b2c61f2b1f3b#r105839060